tests/test_redundant_parentheses.py

```diff
@@ -637,3 +636,32 @@ def test_double_superfluous_but_helping_parentheses_around_bin_op(
         s = f"foo = baz {op2} {parent_expr}"
     s = alteration_func(s)
     assert len(plugin(s)) == 1 + introduced_flakes
+
+
+# GOOD (redundant in slice, but help readability)
+def test_parens_in_slice_1(plugin):
+    s = """foo[i:(i + 1)]
```

Comment:
Of course, this exception should not only work for `+` but for all unary ops, binary ops, boolean operators, etc.